### ISSUE TITLE
Addon stopped working in HA 2021.9

### ISSUE THREAD

**Issue by mmalina** [samueltardieu/homeassistant-addon-pi-cec#3]
This might an issue on HA side, but checking here first.

Today I updated HA to 2021.9.1 and the hdmi_cec switches no longer appear - it worked before the upgrade. I restarted the addon and then also HA just in case, but no luck.

Addon log looks ok. But HA log has this:
```
Error adding entities for domain switch with platform hdmi_cec
Error while setting up hdmi_cec platform for switch
```

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 382, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 587, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 698, in add_to_platform_finish
    self.async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 464, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 498, in _async_write_ha_state
    state = self._stringify_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 470, in _stringify_state
    state = self.state
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 882, in state
    return STATE_ON if self.is_on else STATE_OFF
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 887, in is_on
    return self._attr_is_on
AttributeError: 'CecSwitchEntity' object has no attribute '_attr_is_on'
```

But it does look like a bug in the hdmi_cec integration, right? It's just that in the release notes it didn't mention any changes in it.

**Comment by mmalina** [comment]
I reported it here for HA: https://github.com/home-assistant/core/issues/55649

**Comment by samueltardieu** [comment]
Has it been fixed in HA? I see your issue has been closed after several commits have been made.

**Comment by mmalina** [comment]
Yep, it was fixed in core-2021.9.2 a few days ago. Thanks for checking.
